tests/test_biasadjust.py

```diff
@@ -0,0 +1,345 @@
+                start="2001-01-01",
+                freq="D",
+                as_dataset=True,
+            )
+            .astype("float32")
+            .chunk({"time": -1})
+        )
+        dhist = dsim.sel(time=slice("2001", "2003")).chunk({"time": -1})
+
+        # xscen version
+        dtrain_xscen = xs.train(
+            dref,
+            dhist,
+            var="pr",
+            period=["2001", "2003"],
+            adapt_freq={"thresh": "1 mm d-1"},
+            xclim_train_args={"kind": "*", "nquantiles": 50},
+        )
+
+        out_xscen = xs.adjust(
+            dtrain_xscen,
+            dsim,
+            periods=["2001", "2006"],
+            xclim_adjust_args={
+                "detrend": {
+                    "LoessDetrend": {"f": 0.2, "niter": 1, "d": 0, "weights": "tricube"}
+                },
+                "interp": "nearest",
+                "extrapolation": "constant",
+            },
+        )
+
+        # xclim version
+        with xc.set_options(sdba_extra_output=True):
+            group = xc.sdba.Grouper(group="time.dayofyear", window=31)
+
+            drefx = xc.core.calendar.convert_calendar(
+                dref.sel(time=slice("2001", "2003")), "noleap"
+            )
+            dhistx = xc.core.calendar.convert_calendar(
+                dhist.sel(time=slice("2001", "2003")), "noleap"
+            )
+            dsimx = xc.core.calendar.convert_calendar(
+                dsim.sel(time=slice("2001", "2006")), "noleap"
+            )
+
+            dhist_ad, pth, dP0 = xc.sdba.processing.adapt_freq(
+                drefx["pr"], dhistx["pr"], group=group, thresh="1 mm d-1"
+            )
+
+            QM = xc.sdba.DetrendedQuantileMapping.train(
+                drefx["pr"], dhist_ad, group=group, kind="*", nquantiles=50
+            )
+
+            detrend = xc.sdba.detrending.LoessDetrend(
+                f=0.2, niter=1, d=0, weights="tricube", group=group, kind="*"
+            )
+            out_xclim = QM.adjust(
+                dsimx["pr"], detrend=detrend, interp="nearest", extrapolation="constant"
+            ).rename({"scen": "pr"})
+
+        assert out_xscen.equals(out_xclim)
+
+    # mayeb don't need if we remove the arg ?
+    # def test_window(self):
+    #     dtrain = xs.train(
+    #         self.dref,
+    #         self.dsim.sel(time=slice("2001", "2003")),
+    #         method='ExtremeValues',
+    #         group=False,
+    #         xclim_train_args={'cluster_thresh': "2 degC", "q_thresh": 0.95},
+    #         var="tas",
+    #         period=["2001", "2003"],
+    #     )
+    #
+    #
+    #     out = xs.adjust(self.dtrain,
+    #                     self.dsim,
+    #                     periods=['2001', '2006'],
+    #                     xclim_adjust_args={'scen': scen,
+    #                                        'frac': 0.25},
+    #                     moving_yearly_window={'window': 2, 'step': 1},
+    #                     )
```

Comment:
Still useful?

Reply:
I haven't used that functionality, and reading the Docstrings, I'm not sure what it actually does... Is it there to optimize calculations for big datasets?

In any case, no strong opinion.

Reply:
It's there if you need to do the adjustement on a rolling X-year window (instead of on the whole simulation period like Detrended Quantile mapping). For ExtremeValues in info-crue-cmip6, I construct the window outside of xscen. I think I probably meant to modify it to use xscen eventually, but I never did.

Now, the function has a new name in xclim (`stack_periods`) and so xscen should probably be updated if we want to keep it. But, it now feels like a very specific case  and I am wondering if it needs to be in xscen.

Reply:
Gotcha! Still no strong opinion on it, though, especially if it has never actually been useful.

Although if we decide to keep it, I'd fix it to call the new xclim function.